uint8 uav_id
uint8 wp_id
